test/fuzzing/main.cpp

```diff
@@ -0,0 +1,23 @@
+// if an int main() is needed.
+#if FMT_FUZZ_IMPLEMENT_MAIN
```

Comment:
The macro can probably be dropped and the file conditionally included in the target instead.

Reply:
fixed